lib/readline.js

```diff
@@ -120,8 +119,8 @@ function Interface(input, output, completer, terminal) {
   this.input = input;
   this.historySize = historySize;
   this.removeHistoryDuplicates = !!removeHistoryDuplicates;
-  this.crlfDelay = Math.max(kMincrlfDelay,
-                            Math.min(kMaxcrlfDelay, crlfDelay >>> 0));
+  this.crlfDelay = crlfDelay === Infinity ?
```

Comment:
Actually, instead of fixing the indent just collapse to `Math.max(kMincrlfDelay, crlfDelay);`